README.md

```diff
@@ -174,26 +178,26 @@ Let's look at these findings in details:
 
 ## Step 7
 
-We ask the team to make the necessary changes: `diff step6.yaml step7.yaml`
+We ask the team to make the necessary changes by properly defining the response schema and by adding error status code information. See the differences with this command: `diff step6.yaml step7.yaml`
 
 Let's now evaluate the contract after these changes have been reflected:
 
 ```shell
 spectral lint step7.yaml --verbose --ruleset rulesets/contract.yaml --format pretty
 ```
 
-> Note that the contract ruleset needs to go beyond the provided spectral function.
+> Note that the contract ruleset needs to go beyond the provided spectral function. <!--Why-->
 
```

Comment:
I believe an explanation for why the contract ruleset needs to go beyond the provided spectral function is needed here.